<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertToNgfDialog</class>
 <widget class="QDialog" name="ConvertToNgfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>241</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>416</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>416</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lSourceTable">
       <property name="text">
        <string>Source table</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbSourceTable"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lDestinationTable">
       <property name="text">
        <string>Destination table</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbDestinationTable"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lHeight">
       <property name="text">
        <string>Height of the water</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dsHeight"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lProcessName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbCalculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="hsl">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="hsr">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConvertToNgfDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConvertToNgfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalculate</sender>
   <signal>clicked()</signal>
   <receiver>ConvertToNgfDialog</receiver>
   <slot>launchNextProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>launchNextProcess()</slot>
 </slots>
</ui>
